logstash-core/spec/logstash/util/wrapped_synchronous_queue_spec.rb

```diff
@@ -45,7 +45,7 @@ def take() shift(); end
       def poll(*) shift(); end
     end
 
-    describe "WriteClient | ReadClient" do
+    xdescribe "WriteClient | ReadClient" do
```

Comment:
If you think it's relevant, you can delete these tests instead of skipping them.